### DISCUSSION TITLE
Where to get binaries?

### DISCUSSION THREAD

**Discussion by ghost** [atom-ide-community/atom#413] [General] [answered]
I don't know where to get the binaries

**Comment by ThatXliner** [comment, marked as answer]
We don't have any official releases yet. But there's the ones built via CI. AFAIK so far, we're trying to upgrade stuff, etc.

**Reply by kaosine** [reply]
Yep, right now things are in a massive upgrade process to get things caught up with current node and election at the bare minimum and to upgrade tree-sitter(one of the major core packages)

**Reply by DeeDeeG** [reply]
The binaries from CI are at https://dev.azure.com/atomcommunity/atomcommunity/_build/latest?definitionId=10&branchName=master, find in page "published" (Ctrl + F or Cmd + F). Then click that "8 published" link... An individual binary can be downloaded by pressing the "three dots" button to the side, then "Download artifacts".

**Edit:** Check out autumblazey's awesome custom site (see [comment](https://github.com/atom-community/atom/discussions/413#discussioncomment-3060207) below) that makes it a lot easier to find them!

**Reply by DeeDeeG** [reply]
There were instruction to do this in README.md, but they were moved to here. https://github.com/atom-community/atom/blob/master/docs/Installation.md

I think I prefer them in README.md, personally.

**Reply by meadowsys** [reply]
I did write a little website thing that makes it easier to find binaries [here (https://autumnblazey.github.io/atomcommunity-pipelines/)](https://autumnblazey.github.io/atomcommunity-pipelines/). I know i need to make it more clear but the "Run \<number\>" text is clickable and expands to give some download links (to download directly from azure)

**Comment by lifeisbard** [comment]
I DL'ed the Apple binary and it ran without a hitch, however it's still running on Rosetta. Now that you've upgraded Electron to v12, would you be willing to roll up a version with the universal binary or the dedicated Apple silicon Electron binary so those of us with Apple Silicon may run Atom-Community natively? Several of us have been anxious to use Atom this way. It would be greatly appreciated.

Thank you so much for your attention and please continue the good work.

**Reply by ghost** [reply]
Please do a separate discussion

**Reply by lifeisbard** [reply]
Will do, thank you!

**Reply by lifeisbard** [reply]
New discussion is at [#425](https://github.com/atom-community/atom/discussions/425).
